apps/avatax/src/lib/error-utils.ts

```diff
@@ -43,7 +45,16 @@ export class SubscriptionPayloadErrorChecker {
           },
         );
 
-        this.injectedLogger.error(`Payload contains GraphQL error for ${subscription}`, {
+        if (this.handledErrorPath.every((path) => graphQLError.path?.includes(path))) {
+          // This is handled error - app don't have access to user object. We should migrate clients to use metadata on checkout/order objects instead.
+          this.injectedLogger.info(`Payload contains handled GraphQL error for ${subscription}`, {
```

Comment:
I'd make it warning?

Reply:
I'm thinking if we need this as warning 🤔 This log will be send for every checkout calc taxes call for every client until we remove all `user` field from query. I'm a bit worried if having tones of warnings won't make our logs less readable

Reply:
Fine, we have monitor for it anyway